pgactivity/queries/get_blocking.sql

```diff
@@ -18,7 +18,8 @@ SELECT
       CASE WHEN sq.query LIKE '<IDLE>%%'
           THEN NULL
           ELSE sq.query
-      END AS query
+      END AS query,
+      NULL AS wait
```

Comment:
Why not using `pg_stat_activity.waiting`?

Reply:
no reason .. will fix that way